### PR TITLE
update to relative imports

### PR DESCRIPTION
**Pull request by pq**
🤘 

/cc @devoncarew

### CODE REVIEW

File: lib/src/rules/avoid_catching_errors.dart [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 import 'package:analyzer/dart/ast/ast.dart';
 import 'package:analyzer/dart/ast/visitor.dart';
-import 'package:linter/src/analyzer.dart';
-import 'package:linter/src/util/dart_type_utilities.dart';
+import '../analyzer.dart';
```
Comment (devoncarew) [inline lib/src/rules/avoid_catching_errors.dart]: organize imports?

File: lib/src/rules/avoid_catches_without_on_clauses.dart [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 import 'package:analyzer/dart/ast/ast.dart';
 import 'package:analyzer/dart/ast/visitor.dart';
-import 'package:linter/src/analyzer.dart';
+import '../analyzer.dart';
```
Comment (devoncarew) [inline lib/src/rules/avoid_catches_without_on_clauses.dart]: organize imports?

File: lib/src/rules/avoid_bool_literals_in_conditional_expressions.dart [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 import 'package:analyzer/dart/ast/ast.dart';
 import 'package:analyzer/dart/ast/visitor.dart';
-import 'package:linter/src/analyzer.dart';
+import '../analyzer.dart';
```
Comment (devoncarew) [inline lib/src/rules/avoid_bool_literals_in_conditional_expressions.dart]: organize imports?

**Comment by pq** [comment]
> It looks like the CI found two files that need formatting.

Ironically, one of them was the format validation code itself!

image

😬

**Pull request merged**